src/diffpy/labpdfproc/functions.py

```diff
@@ -191,14 +191,14 @@ def _cve_brute_force(diffraction_data, mud):
     muls = np.array(muls) / abs_correction.total_points_in_grid
     cve = 1 / muls
 
-    cve_do = Diffraction_object(wavelength=diffraction_data.wavelength)
-    cve_do.insert_scattering_quantity(
-        TTH_GRID,
-        cve,
-        "tth",
-        metadata=diffraction_data.metadata,
-        name=f"absorption correction, cve, for {diffraction_data.name}",
+    cve_do = DiffractionObject(
```

Comment:
this feels better to me.  I think this change we made in the DO was probably a good idea.....